src/fiatExchanges/PaymentMethodSection.tsx

```diff
@@ -0,0 +1,245 @@
+              ...styles.expandableContainer,
+              paddingVertical: isExpandable ? (expanded ? 22 : 27) : 16,
+            }}
+            isExpandable={isExpandable}
+            isExpanded={expanded}
+          >
+            {isExpandable ? (
+              <>
+                <View style={styles.left}>
+                  <Text style={styles.category}>
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.card')
+                      : t('selectProviderScreen.bank')}
+                  </Text>
+                  {!expanded && (
+                    <Text style={styles.fee}>
+                      {renderFeeAmount(sectionQuotes[0].quote, t('selectProviderScreen.minFee'))}
+                    </Text>
+                  )}
+                </View>
+
+                <View style={styles.right}>
+                  <Text style={styles.providerDropdown}>
+                    {t('selectProviderScreen.numProviders', { count: sectionQuotes.length })}
+                  </Text>
+                </View>
+              </>
+            ) : (
+              <>
+                <View style={styles.left}>
+                  <Text style={styles.category}>
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.card')
+                      : t('selectProviderScreen.bank')}
+                  </Text>
+                  <Text style={styles.fee}>
+                    {renderFeeAmount(sectionQuotes[0].quote, t('selectProviderScreen.fee'))}
+                  </Text>
+                  <Text style={styles.topInfo}>
+                    {t('selectProviderScreen.idRequired')} |{' '}
+                    {paymentMethod === PaymentMethod.Card
+                      ? t('selectProviderScreen.oneHour')
+                      : t('selectProviderScreen.numDays')}
+                  </Text>
+                </View>
+
+                <View style={styles.imageContainer}>
+                  <Image
+                    testID={`image-${sectionQuotes[0].provider.name}`}
+                    source={{ uri: sectionQuotes[0].provider.logoWide }}
+                    style={styles.providerImage}
+                    resizeMode="center"
+                  />
+                </View>
+              </>
+            )}
+          </Expandable>
+        </View>
+      </Touchable>
+      {expanded &&
+        sectionQuotes.map((cicoQuote, index) => (
+          <Touchable onPress={(quoteOnPress(cicoQuote) as unknown) as () => void}>
+            <View style={styles.expandedContainer}>
+              <View style={styles.left}>
+                <Text style={styles.expandedFee}>
+                  {renderFeeAmount(cicoQuote.quote, t('selectProviderScreen.fee'))}
+                </Text>
+                <Text style={styles.expandedInfo}>
+                  {t('selectProviderScreen.idRequired')} |{' '}
+                  {paymentMethod === PaymentMethod.Card
```

Comment:
Looks like there's some duplicated logic for displaying settlement times between here and line 127... would be good to deduplicate if possible, though nbd